libs/fixtures/data/electionMinimalExhaustiveSample/cvrFiles/cdf_cast_vote_record_report.json

```diff
@@ -0,0 +1,375 @@
+{
+  "@type": "CVR.CastVoteRecordReport",
+  "Version": "1.0.0",
+  "ReportType": "originating-device-export",
+  "ReportingDevice": [
+    {
+      "@type": "CVR.ReportingDevice",
+      "SerialNumber": "scanner-1"
+    }
+  ],
+  "ReportGeneratingDeviceIds": ["scanner-1"],
+  "Party": [
+    {
+      "@type": "CVR.Party",
+      "@id": "0",
+      "Name": "Mammal Party",
+      "Abbreviation": "Ma"
+    },
+    {
+      "@type": "CVR.Party",
+      "@id": "1",
+      "Name": "Fish Party",
+      "Abbreviation": "F"
+    }
+  ],
+  "Election": [
+    {
+      "@type": "CVR.Election",
+      "@id": "some-id",
```

Comment:
I don't think this `id` matters, could be anything. We don't currently have an election ID on our CVRs.

Reply:
Will just use election hash!